grammars/julia.cson

```diff
@@ -170,7 +170,7 @@ repository:
         name: "storage.modifier.variable.julia"
       }
       {
-        match: "(@[[:alpha:]_][[:word:]!]*)(?=[ (])"
+        match: "(@[[:alpha:]_][[:word:]!]*)"
```

Comment:
@garborg I think you might have a better idea for why the `[ (]` was here in the first place. Any ideas